.github/workflows/push.yml

```diff
@@ -124,6 +124,10 @@ jobs:
     env:
       MODIN_ENGINE: ${{matrix.engine}}
       MODIN_MEMORY: 1000000000
+      # Only test readin from SQL server on ubuntu for now. Eventually, we
```

Comment:
ditto, should this be "reading" instead of "readin"?

Reply:
```suggestion
      # Only test reading from SQL server on ubuntu for now. Eventually, we
```